klog.go

```diff
@@ -914,9 +917,9 @@ func (l *loggingT) output(s severity, log logr.Logger, buf *buffer, file string,
 		// TODO: set 'severity' and caller information as structured log info
 		// keysAndValues := []interface{}{"severity", severityName[s], "file", file, "line", line}
 		if s == errorLog {
-			l.logr.Error(nil, string(data))
```

Comment:
That the old code was calling `l.logr` instead of its `log` parameters looks like a bug to me.